scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Scripts
+
+- `create_service.sh`
+  - creates managed instance group and backend service
+- `create_health_check.sh`
+  - creates health check
+- `config_traffic_director.sh`
+  - creates traffic director configutation
```

Comment:
Similar.  What kind of traffic director configuration?  What does it do?  Why do I want it?

Reply:
Done